src/K1DModel/parameters.jl

```diff
@@ -1,25 +1,62 @@
-"""
-    Create Kinematic1D model parameters.
-
-    (Overwriting the default values happens in the driver file.
-    Here we are just creating the Kinematic1DParameters struct)
-"""
 module Parameters
 
+using DocStringExtensions
+import ClimaParams as CP
+
 abstract type AbstractKinematicParameters end
 const AKP = AbstractKinematicParameters
 
-# Define KiD parameters
+"""
+    Kinematic1DParameters{FT}
+
+    Free parameters for the kinematic 1-dimensional simulation
+
+    #Fields
+    $(DocStringExtensions.FIELDS)
+"""
 Base.@kwdef struct Kinematic1DParameters{FT} <: AKP
+    "Maximum updraft momentum flux m/s kg/m3"
     w1::FT
+    "Time when the updraft is switched off [s]"
     t1::FT
+    "Surface pressure [Pa]"
     p0::FT
+    "Switch to include flux correction for moisture transport"
     qtot_flux_correction::Int
+    "Assumed aerosol dry radius [m]"
     r_dry::FT
+    "Assumed aerosol standard deviation [-]"
     std_dry::FT
+    "Assumed aerosol hygroscopicity"
     κ::FT
 end
 
+function Kinematic1DParameters(td::CP.AbstractTOMLDict)
+    #name_map = (;
+    #    :prescribed_flow_w1 => :w1,
+    #    :prescribed_flow_t1 => :t1,
+    #    :surface_pressure => :p0,
+    #    :qtot_flux_correction_flag => :qtot_flux_correction,
+    #    :r_dry => :r_dry,
+    #    :std_dry => :std_dry,
+    #    :kappa => :κ,
+    #)
+    #parameters = CP.get_parameter_values(td, name_map, "Kinematic1D")
+    #FT = CP.float_type(td)
+    #return Kinematic1DParameters{FT}(; parameters)
+    (; data) = td
+    FT = CP.float_type(td)
+    return Kinematic1DParameters(
+        FT(data["prescribed_flow_w1"]["value"]),
+        FT(data["prescribed_flow_t1"]["value"]),
+        FT(data["surface_pressure"]["value"]),
+        Int(data["qtot_flux_correction_flag"]["value"]),
```

Comment:
@nefrathenrici  - Is there a better way to handle a bool?

Reply:
Is qtot_flux_correction_flag meant to be a Bool or an Int?
 ClimaParams supports bool types, so reading it in using the API will parse it as a Julia Boolean if the TOML type = bool